server.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+console.log('hello my bitches');
+
+const express = require('express');
+
+const app = express();
+
+const PORT = 3002
+
+const weatherData = require('./data/weather.json');
+
+app.get('/', (request, response) => {
+  response.send('hello, from our server, bitches');
+})
+
+app.get('/throw-an-error', (request, response) => {
+
+  throw 'you not good!'
+})
+
+
+app.get('/weather', (request, response) => {
+  let city_name = request.query.city_name;
+  console.log(city_name);
+  response.send(weatherData.filter(weather => weather.city_name === city_name).map(weather => new Weather(weather)));
```

Comment:
- Task card two asks us to use `find()` to discover which city `lat, lon and searchQuery` belong to. Then return an error if user did not search for one of the cities we have info about 👍

Reply:
Please ignore my comment for find(), I spoke with Ryan and he told me that he said you guys could use filter (you will get one item this way) but both were demoed so you can use either👍